filebeat/module/postgresql/log/test/postgresql-9.6-multi-core.log

```diff
@@ -0,0 +1,19 @@
+2018-11-17 13:14:01 CEST [12975-1] [unknown]@[unknown] LOG:  incomplete startup packet
```

Comment:
Add the expected file for this one please, I [mentioned](https://github.com/elastic/beats/pull/9431/files#r240010463) to remove the other one that was not related to this change.